### DISCUSSION TITLE
Overwrite property datatype

### DISCUSSION THREAD

**Discussion by kirillsbarsukov** [MichalLytek/typegraphql-prisma#330] [Q&A] [answered]
Hi all!

I have a question.  This is my schema.prisma. Pretty simple
> model giftcard {
  giftcardid        BigInt    @id
}
When I generate graphql, I got this
> id: BigInt!

Is there any way to change BigInt to ID ?
> id: ID!

Thank you!

**Comment by MichalLytek** [comment, marked as answer]
There is a `emitIdAsIDType` generator option. Looks like it's not well described in docs 😕

**Reply by kirillsbarsukov** [reply]
Thank you for your answer. I made [PR](https://github.com/MichalLytek/typegraphql-prisma/pull/331). My 2 cents.
